tests/unit/test_model.py

```diff
@@ -869,3 +874,24 @@ def test_validate_single_required_field_with_bad_values_raises_value_error(
 
 def test_validate_single_required_field_with_only_one_field_does_not_raise_value_error():
     validate_single_required_field(["One", "Two", "Three"], [None, None, 3])
+
+
+def test_atlan_tag_names(monkeypatch):
+    tag_name = "Issue"
+    tag_id = "123"
+
+    def get_name_for_id(value):
+        if value == tag_id:
+            return tag_name
+        return ""
+
+    monkeypatch.setattr(
+        pyatlan.cache.atlan_tag_cache.AtlanTagCache,
+        "get_name_for_id",
+        get_name_for_id,
+    )
+
+    referenceable = Referenceable()
+    referenceable.classification_names = [tag_id, "456"]
```

Comment:
Is there a positive test here, too (`123` matching a defined tag and thus getting back `Issue` as a name)? 

This only looks like the negative test (`456` not matching any defined tag, and thus getting back the deleted sentinel)

Reply:
Yes the assert on line 897 says that referenceable.atlan_tag_names == [tag_name, DELETED_]. tag_name is "Issue"

Reply:
Ah, now I see it — eyes are blurry from staring at a screen for too long 😵‍💫